bf-interpreter/main.py

```diff
@@ -0,0 +1,208 @@
+"""Simple brainfuck interpreter.
```

Comment:
This string is shown to users of the badge store, whether they install it or not, and seems incompatible with the first of the [badge store rules](https://badge.emfcamp.org/wiki/TiLDA_MK4/Badge_Store_Submissions):
> App has to be "Rated G: General Audiences – all ages admitted"

Reply:
Completely fair point, would `Simple brainf*** interpreter` be acceptable? I figured "BF" might be ambiguous on its own, but please change it to whatever you feel is appropriate.